### DISCUSSION TITLE
install command report: both %LOCALAPPDATA% and %APPDATA% were unreadable

### DISCUSSION THREAD

**Discussion by imxtx** [microsoft/vcpkg#33654] [Q&A]
![image](https://github.com/microsoft/vcpkg/assets/93981430/66b108bd-6ddd-4a75-bdfc-eea83a8aac7e)

**Comment by autoantwort** [comment]
Are you able to open the folders in the file explorer without admin rights?

**Comment by imxtx** [comment]
Yes I'm able to open them without admin rights. 
    
        
        
            
                
                    Tianxin Xie
                    
                    
                                
                                    ***@***.***
                                
                
            
        
        
    

---- Replied Message ***@***.***>Date09/09/2023 01:17 ***@***.***> CcT.X. ***@***.***>***@***.***>SubjectRe: [microsoft/vcpkg] install command report: both %LOCALAPPDATA% and %APPDATA% were unreadable (Discussion #33654)
Are you able to open the folders in the file explorer without admin rights?

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/microsoft/vcpkg/discussions/33654#discussioncomment-6950659",
"url": "https://github.com/microsoft/vcpkg/discussions/33654#discussioncomment-6950659",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
